test/e2e/testing-manifests/statefulset/mysql-upgrade/statefulset.yaml

```diff
@@ -139,10 +139,9 @@ spec:
   volumeClaimTemplates:
   - metadata:
       name: data
-      annotations:
-        volume.beta.kubernetes.io/storage-class: default
     spec:
       accessModes: ["ReadWriteOnce"]
+      storageClassName: default
```

Comment:
I replace the beta annotations with spec field in the example. I verified on my env, it worked.
cc @liggitt @msau42